rendermodules/em_montage_qc/detect_montage_defects.py

```diff
@@ -362,9 +362,12 @@ def run(self):
         centroids = [seam_centroids[i] for i in seams_indices]
 
         self.args['output_html'] = self.args['out_html_dir']
-        if len(combinedz) > 0:
-            if self.args['plot_sections']:
-                self.args['output_html'] = plot_section_maps(self.render, new_poststitched, combinedz, out_html_dir=self.args['output_html'])
+        if self.args['plot_sections']:
+            self.args['output_html'] = plot_section_maps(self.render, self.args['poststitched_stack'], post_tspecs, disconnected_tiles, gap_tiles, seam_centroids, zvalues, out_html_dir=self.args['output_html'])
```

Comment:
I'm confused by the use of the `output_html` and `out_html_dir` here -- the data types change a lot in these few lines and ultimately end up invalidating the schema -- are these used somewhere where they need to be attached to args?

Reply:
changing this to avoid confusion in the schema... the older version of which had out_html_dir and output_html as Strings